<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QMainWindow" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HELIACAL EXCHANGE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #10161f;
color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #10161f;
color: white;</string>
   </property>
   <widget class="QLineEdit" name="lineEditName">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>250</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 22px; 
   padding-left:60px; 
background-color:#1d2633;
color: rgb(255, 255, 255);
    font-size: 14px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_phone">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>490</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;
font-weight: bold;
background-color:#1d2633;
color: white;
border-radius: 22px; 
padding: 0px 12px; 

</string>
    </property>
    <property name="text">
     <string>           phone:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>650</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">            color: white;
            border-radius: 10px;
            font-size: 16px;
           border: none;  
background-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>register</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_copas">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>410</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;
font-weight: bold;
background-color:#1d2633;
color: white;
border-radius: 22px; 
padding: 0px 12px; 

</string>
    </property>
    <property name="text">
     <string>confirm pwd : *</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_password">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>330</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;
font-weight: bold;
background-color:#1d2633;
color: white;
border-radius: 22px; 
padding: 0px 12px; 

</string>
    </property>
    <property name="text">
     <string>  password :  *</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditEmail">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>170</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 22px; 
   padding-left:60px; 
background-color:#1d2633;
color: rgb(255, 255, 255);
font-size: 14px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>40</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">            color: white;
            border-radius: 10px;
            font-size: 36px;
           border: none;  </string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_invitationcode">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>570</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;
font-weight: bold;
background-color:#1d2633;
color: white;
border-radius: 22px; 
padding: 0px 12px; 

</string>
    </property>
    <property name="text">
     <string>invitationcode:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_gosignin">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>120</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 0);
            color: white;
            border-radius: 10px;
            font-size: 16px;
           border: none;  </string>
    </property>
    <property name="text">
     <string>Sign in</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_email">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>170</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;
font-weight: bold;
background-color:#1d2633;
color: white;
border-radius: 22px; 
padding: 0px 12px; 

</string>
    </property>
    <property name="text">
     <string>        Email :  *</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>120</y>
      <width>241</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color: rgb(255, 255, 255);
    font-size: 18px;

</string>
    </property>
    <property name="text">
     <string>Have an account? Sign in now</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPhone">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>490</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 22px; 
   padding-left:60px; 
background-color:#1d2633;
color: rgb(255, 255, 255);
 font-size: 14px;
</string>
    </property>
    <property name="placeholderText">
     <string>    optional</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>720</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirmpassword">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>410</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 22px; 
   padding-left:60px; 
background-color:#1d2633;
color: rgb(255, 255, 255);
    font-size: 14px;
</string>
    </property>
    <property name="text">
     <string>Abbas1383</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>250</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;
font-weight: bold;
background-color:#1d2633;
color: white;
border-radius: 22px; 
padding: 0px 12px; 

</string>
    </property>
    <property name="text">
     <string>User Name:  *</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPassword">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>330</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 22px; 
   padding-left:60px; 
background-color:#1d2633;
color: rgb(255, 255, 255);
    font-size: 14px;
</string>
    </property>
    <property name="text">
     <string>Abbas1383</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="invitationcode">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>570</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 22px; 
   padding-left:60px; 
background-color:#1d2633;
color: rgb(255, 255, 255);
    font-size: 14px;
</string>
    </property>
    <property name="placeholderText">
     <string>     optional</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>881</width>
      <height>701</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">    border-radius: 8px; 
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>790</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">            color: white;
            border-radius: 10px;
            font-size: 16px;
           border: none;  
background-color: rgb(125, 0, 0);</string>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <zorder>lineEditName</zorder>
   <zorder>pushButton</zorder>
   <zorder>lineEditEmail</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>pushButton_gosignin</zorder>
   <zorder>lineEdit</zorder>
   <zorder>lineEditPhone</zorder>
   <zorder>confirmpassword</zorder>
   <zorder>lineEditPassword</zorder>
   <zorder>invitationcode</zorder>
   <zorder>label_6</zorder>
   <zorder>label_name</zorder>
   <zorder>label_email</zorder>
   <zorder>label_password</zorder>
   <zorder>label_copas</zorder>
   <zorder>label_invitationcode</zorder>
   <zorder>label_phone</zorder>
   <zorder>exit_btn</zorder>
   <zorder>textEdit</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>confirmpassword</tabstop>
  <tabstop>lineEditPhone</tabstop>
  <tabstop>invitationcode</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>pushButton_gosignin</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>exit_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
